File: Modul4/PRAK405-2310817110008-Muhammad_Raihan.c
#include <stdio.h>

void main ()
{
    int n,k,i,j,s,hasil=0,total=0;
    scanf("%d %d", &n,&k);
    for(i=1; i<=n; i++) {
        for(j=i; j>=1; j--) {
            printf("(%d * %d)", j,k);
            if (j>1) {
                printf(" + ");
            }
            else {
                printf(" = ");
            }
            if (j==i) {
                s = j*k;
                hasil = hasil + s;
            }
        }
        printf("%d\n", hasil);
        total = total + hasil;
    }
    printf("%d", total);
}

File: Modul4/PRAK405-2310817110008-Muhammad_Raihan.py
n,k = map(int, input().split())
hasil=0; total=0
for i in range(1, n+1, 1):
    for j in range(i, 0, -1):
        print("({} * {})".format(j,k),end=' ')
        if j>1:
            print("+",end=' ')
        else:
            print("=",end=' ')
        if j==i:
            s = j*k
            hasil = hasil+s
    print(hasil)
    total = total+hasil
print(total)